deployments/ecs/agent.yaml

```diff
@@ -56,6 +57,19 @@ monitors:
     # config for each EC2 instance.
   - {"#from": "env:EXTRA_MONITOR_CONFIG", default: [], flatten: true}
 
+  # Enables the Smart Agent to receive traces.
+  - type: signalfx-forwarder
+    # Specify the URL:port where the Smart Agent listens for traces.
+    listenAddress: "0.0.0.0:9080/v1/trace"
+    # Optionally, add span tags to each span in a trace the Smart Agent receives.
+    defaultSpanTags:
```

Comment:
I would also comment out these lines as it prevents being able to refer to this agent.yaml in a demo without making these span tags show up.

Reply:
Otherwise looks good.